.ci/Jenkinsfile

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env groovy
 
-@Library('apm@current') _
+@Library('apm@test/with-docker-login') _
```

Comment:
Will update this once the shared library gets released including https://github.com/elastic/apm-pipeline-library/pull/1306